pypeit/display/display.py

```diff
@@ -250,6 +250,49 @@ def show_image(inp, chname='Image', waveimg=None, bitmask=None, mask=None, exten
 
     return viewer, ch
 
+
+def show_points(viewer, ch, spec, spat, color='cyan', legend=None, legend_spec=None, legend_spat=None):
+    """
+
+
+
+    Parameters
+    ----------
+    viewer:
```

Comment:
This is a mix of Google and numpy doc format.

Reply:
Fixed.